api/app/settings/common.py

```diff
@@ -617,8 +617,10 @@
 
 ENABLE_DB_LOGGING = env.bool("DJANGO_ENABLE_DB_LOGGING", default=False)
 if ENABLE_DB_LOGGING:
-    if not DEBUG:  # pragma: no cover
-        raise ImproperlyConfigured("DB logging will not work without DEBUG=True")
+    if not DEBUG:
+        warnings.warn("Setting DEBUG=True to ensure DB logging functions correctly.")
+        DEBUG = True
```

Comment:
Not a fan of re-setting `DEBUG`. I think this is better:
 
```python
DEBUG = env.bool("DEBUG", default=False) or env.bool("DJANGO_ENABLE_DB_LOGGING", default=False)
```

Reply:
I don't think merging them is a good idea... `DB_LOGGING` is very verbose

Reply:
Agreed